src/pages/notFound.vue

```diff
@@ -0,0 +1,25 @@
+<template>
+    <div v-if="appConfig" class="not-found ma-4 text-body-1">
+        <p>Sorry, this page does not exist!</p>
+        <p v-if="detail">
+            {{ detail }}
+        </p>
+        <p>
+            Go to the <router-link to="/">{{ appConfig.title }} home page</router-link> instead.
```

Comment:
When I follow this link from `https://mrc-ide.github.io/arbomap/dengue/asdf/FOI`, the browser back button doesn't work, because it takes me back to the homepage `https://mrc-ide.github.io/arbomap/dengue/may24/` which then redirects to `https://mrc-ide.github.io/arbomap/dengue/may24/FOI`.

Reply:
It seems like there is a redirect happening which also populates the browser history.

Reply:
My browser history gets trapped in the loop if I visit
https://mrc-ide.github.io/arbomap
or
https://mrc-ide.github.io/arbomap/dengue/may24
because these both cause a redirect which the back button can't escape from. This doesn't need to be fixed in this PR, it could be a new ticket to address later.